src/modm/ui/menu/abstract_menu.hpp

```diff
@@ -30,12 +30,17 @@ namespace modm{
 	 * \author Thorsten Lajewski
 	 * \ingroup modm_ui_menu
 	 */
-	class AbstractMenu: public AbstractView
+	template<typename Allocator = allocator::Dynamic<IAbstractView> >
```

Comment:
Could you check if we can instead use `std::allocator`? I think `modm::allocator` was just for AVRs, but now we have a libstdc++ for it.

Reply:
I'm not allowed to push changes to this branch:
```
 $ git push uw-advanced-robotics
ERROR: Permission to uw-advanced-robotics/modm.git denied to salkinium.
```

So please [cherry-pick this commit over to your branch](https://github.com/salkinium/modm/commit/3ccda11c0c227688963d17af1330bae0401da7d4) and let's see if the CI passes.